ubuntu22.04/nvidia-driver

```diff
@@ -73,14 +76,10 @@ _install_prerequisites() (
     mkdir -p /lib/modules/${KERNEL_VERSION}/proc
 
     echo "Installing Linux kernel headers..."
-    apt-get -qq install --no-install-recommends linux-headers-${KERNEL_VERSION} > /dev/null
+    #apt-get -qq install --no-install-recommends linux-headers-${KERNEL_VERSION} > /dev/null
 
     echo "Installing Linux kernel module files..."
     apt-get -qq download linux-image-${KERNEL_VERSION} && dpkg -x linux-image*.deb .
-    { apt-get -qq download linux-modules-${KERNEL_VERSION} && dpkg -x linux-modules*.deb . || true; } 2> /dev/null
-    mv lib/modules/${KERNEL_VERSION}/modules.* /lib/modules/${KERNEL_VERSION}
-    mv lib/modules/${KERNEL_VERSION}/kernel /lib/modules/${KERNEL_VERSION}
-    depmod ${KERNEL_VERSION}
```

Comment:
Since re-using the kernel headers from the host is a separate feature, let's make this change in a follow-up PR.